charts/rbacdefinitions/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: rbacdefinitions
```

Comment:
can we have dash in the name please?
```suggestion
name: rbac-definitions
```

Reply:
yeah was bugging me too during the whole creation. glad you think the same. I am changing this later.

Reply:
i was even considering `rbac-manager-definitions` but it might get a tad to long with that.